src/Routers/FeaturesRouter.js

```diff
@@ -20,10 +20,10 @@ export class FeaturesRouter extends PromiseRouter {
     if (!req.auth.isMaster) {
       return masterKeyRequiredResponse();
     }
-    
+
     return Promise.resolve({
       response: {
-        results: [getFeatures()]
+        results: getFeatures()
```

Comment:
In regards to `code`/`message` - we shouldn't be relying on keys in the body for success/failure, but on http status code. If >=400 - then the request failed and any info in the body is describing the error.